LoopUITests/Screens/BaseScreen.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  BaseScreen.swift
+//  LoopUITests
+//
+//  Created by Ginny Yadav on 10/27/23.
+//  Copyright © 2023 LoopKit Authors. All rights reserved.
+//
+
+import XCTest
+
+class BaseScreen {
+    var app: XCUIApplication
+    var springboardApp: XCUIApplication
+    var bundleIdentifier: String?
+
+    init(app: XCUIApplication) {
+        self.app = app
+        self.springboardApp = XCUIApplication(bundleIdentifier:"com.apple.springboard")
+        self.bundleIdentifier = Bundle.main.bundleIdentifier
+    }
+    
+    func deleteApp() {
+        XCUIApplication().terminate()
+
+        let icon = springboardApp.icons["Tidepool Loop"]
```

Comment:
This is testing the Tidepool Loop version of Loop, but this is in shared code.  We may need to find a way to make this not Tidepool Specific

Reply:
We can derive this. Will update.